JavaWeb3/src/main/resources/templates/sign_up_vue.html

```diff
@@ -1,138 +1,145 @@
-                        email: '',
-                        name: '',
-                        pwd: '',
-                        picture: ''
-                    },
-                    rules: {
-                        email: [
-                          { required: true, message: 'Please enter your email address.', trigger: 'blur' }
-                        ],
-                        name: [
-                          { required: true, message: 'Please enter your user name.', trigger: 'blur' },
-                          { max: 12, message: 'No more than 12 characters.', trigger: 'blur' }
-                        ],
-                        pwd: [
-                          { required: true, message: 'Please enter your password.', trigger: 'blur' }
-                        ]
-                    }
-	            };
-            },
-            mounted() {
-                if ('WebSocket' in window) {
-                    this.websocket = new WebSocket('ws://localhost:1234');
-                    this.initWebSocket();
+<head>
+	<meta charset="utf-8">
+	<title>Sign Up</title>
+	<link rel="stylesheet" type="text/css" href="../static/style/theme/index.css"/>
+	<script type="text/javascript" src="../static/js/vue.js"></script>
+	<script type="text/javascript" src="../static/js/md5.js"></script>
+	<!-- import JavaScript -->
+	<script src="https://unpkg.com/element-ui/lib/index.js"></script>
+</head>
+<body>
+<div id="app" :style='divStyle'>
+	<el-form :model="ruleForm" :rules="rules" ref="ruleForm" label-width="150px" class="demo-ruleForm">
+		<el-form-item label="Email Address" prop="email">
+			<el-input v-model="ruleForm.email"></el-input>
+		</el-form-item>
+		<el-form-item label="User Name" prop="name">
+			<el-input v-model="ruleForm.name"></el-input>
+		</el-form-item>
+		<el-form-item label="Password" prop="pwd">
+			<el-input v-model="ruleForm.pwd" show-password></el-input>
+		</el-form-item>
+		<el-form-item>
+			<el-button type="primary" @click="submitForm()">Sign Up</el-button>
+			<el-button @click="resetForm('ruleForm')">Reset</el-button>
+			<el-button @click="onSignIn">Sign In</el-button>
+		</el-form-item>
+	</el-form>
+	<div>{{request}}</div>
+	<div>{{response}}</div>
+	<div>{{parsed_response}}</div>
+</div>
+</body>
+
+<script>
+    new Vue({
+        el: '#app',
+        data: function() {
+            return {
+                websocket: null,
+                request: "",
+                response: "",
+                parsed_response: "",
+                divStyle: "position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%); width: 500px; height: 300px;",
+                ruleForm: {
+                    email: '',
+                    name: '',
+                    pwd: '',
+                    picture: ''
+                },
+                rules: {
+                    email: [
+                        { required: true, message: 'Please enter your email address.', trigger: 'blur' }
```

Comment:
I will add a wiki page for **input validations** for the `email`, `name` and `password` that we agreed upon. In the next PR, we will make sure that they correspond to both client applications.